### PR TITLE
[master] Fix issue with fc_eltwise fusing

### PR DESCRIPTION
**Pull request by piotrwolinski-intel**
These changes are introduced to allow more activation functions to be used with fused and quantized fc_eltwise.

### CODE REVIEW

File: src/operator/nn/dnnl/dnnl_act.cc [hunk 1]
```diff
@@ -70,9 +70,9 @@ bool SupportDNNLLeakyRelu(const LeakyReLUParam& param, const NDArray& input) {
 }
 
 bool SupportQuantizedDNNLAct(const ActivationParam& param) {
-  // TODO(zhennan): Add more activation type when dnnl supports.
-  //                Remove this when it's identity to SupportDNNLAct.
-  return param.act_type == activation::kReLU;
+  // Although it is the same as SupportDNNLAct i left it here, so when new activations
```
Comment (bartekkuncer) [inline src/operator/nn/dnnl/dnnl_act.cc:73]: I understand what you mean but would still prefer this removed. I believe that a developer will create separate function for more sophisticated functions if needed.

**Review by bartekkuncer — APPROVED** [review]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [unix-cpu, windows-gpu]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [website, unix-gpu, unix-cpu, miscellaneous]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [unix-cpu, centos-cpu]

**Review by anko-intel — APPROVED** [review]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [unix-cpu, centos-gpu, unix-gpu, windows-gpu, website]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [centos-gpu, unix-gpu, website]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [centos-gpu]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [centos-gpu]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [windows-gpu]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [windows-gpu]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [clang, miscellaneous, unix-gpu, website]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [centos-gpu, unix-cpu, windows-gpu]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [centos-gpu]

**Comment by piotrwolinski-intel** [comment]
@mxnet-bot run ci [centos-gpu]

**Review by bgawrych — APPROVED** [review]

**Pull request merged**